src/amo/components/ShowMoreCard/index.js

```diff
@@ -98,9 +80,11 @@ export class ShowMoreCardBase extends React.Component<InternalProps> {
   }
 
   resetUIState() {
-    this.props.setUIState({
-      ...initialUIState,
-    });
+    if (this.props.uiState !== initialUIState) {
```

Comment:
Is this still needed?

Reply:
I think so, in the case where the content isn't being truncated, but I can re-check.

Reply:
It doesn't seem to have an effect anymore, so it can be removed. I'll do so.